runtime/src/handler.rs

```diff
@@ -103,7 +103,7 @@ pub trait Handler {
 		value: U256,
 		init_code: Vec<u8>,
 		target_gas: Option<u64>,
-	) -> Capture<(ExitReason, Option<H160>, Vec<u8>), Self::CreateInterrupt>;
+	) -> Capture<(ExitReason, Vec<u8>), Self::CreateInterrupt>;
```

Comment:
This is technically a breaking change from the perspective of sputnikvm as a library since we are changing the signature of a method on a public trait. This is not an issue, especially if we are not planning on ever syncing up with upstream again and we are the only users of the library. But I thought it is worth pointing out just in case.

Reply:
Yes, it's breaking changes. With the release of `Prague`, we plan to de-fork into our own library `aurora-evm`, since upstream has completely abandoned the previous version and moved to the new version v1.0-alpha. Moreover, they do not have any development at all for a long time. At the moment, our fork is so different that it is ALREADY practically an independent library. ✊